### PR TITLE
New package: shirowanisan.COEIROINK version 2.2.1

### PR DESCRIPTION
**Pull request by Exorcism0666**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/133861)

### CODE REVIEW

**Comment by Exorcism0666** [comment]
## For moderators

This account is automated by Github Actions and the source code was created by CoolPlayLin. If you have any questions about any pull request, don't hesitate to ping @Exorcism0666, I'll get a notification.
>[!important]
>Please carefully review these Pull Request before merging. If it is a Pull Request for removing incorrect content and the URLs are issue, free upon manual checking, please close this Pull Request directly. (It is best to inform Exorcism0666 of the closure.)

良い一日をお過ごしください！

**Comment by stephengillie** [comment]
The file `COEIROINK_WIN_CPU_v.2.2.1.exe` appears to be a self-extracting ZIP file.

File: manifests/s/shirowanisan/COEIROINK/2.2.1/shirowanisan.COEIROINK.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Created with Komac v1.11.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: shirowanisan.COEIROINK
+PackageVersion: 2.2.1
+InstallerType: zip
+NestedInstallerType: portable
```
Comment (stephengillie) [inline manifests/s/shirowanisan/COEIROINK/2.2.1/shirowanisan.COEIROINK.installer.yaml:7]:
```suggestion
NestedInstallerType: exe
```

**Comment by stephengillie** [comment]
With the updated NestedInstallerType, this application installs and launches normally on a Windows 10 VM. But it gives this message when exiting:
image

(Internet translation): Confirm Completion
1) Do you want to close the application?
All in-progress work may be lost when you exit.
-> Cancel
-> End

**Review by stephengillie — APPROVED** [review]

**Pull request merged**